docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+        </orderedlist>
+
+        <para>Once Terraform successfully applies all the modules in the
+        appropriate sequence, they then initialize and enter the ready state,
+        your HPCC Systems cluster is up and running.</para>
+      </sect3>
+
+      <sect3>
+        <title>Verify the Installation</title>
+
+        <para>With your successful Terraform deployment Kubernetes has
+        provisioned all the required HPCC Systems pods. To check their status
+        run:</para>
+
+        <programlisting>kubectl get pods </programlisting>
+
+        <para><emphasis role="bold">Note</emphasis>: If this is the first time
+        helm install has been run, it may take some time for the pods to all
+        get into a <emphasis>Running</emphasis> state. Azure will need to pull
+        container images from Docker, bring each component online, etc. Once
+        all the pods are running, the HPCC Systems cluster is ready to be
+        used.</para>
+
+        <para><graphic
+        fileref="../../images/CNTR_kubctlGetPodsRunning.jpg" />Once all the
+        pods are ready and in the Running state, then you can use your HPCC
+        Systems container cluster.</para>
+      </sect3>
+    </sect2>
+
+    <sect2 id="Az-AccessECLWatch">
+      <title>Accessing ECLWatch</title>
+
+      <para>To access ECLWatch, an external IP for the ESP running ECLWatch is
+      required. This will be listed as the <emphasis>eclwatch</emphasis>
+      service, and can be obtained by running the following command:</para>
+
+      <programlisting>kubectl get svc </programlisting>
+
+      <para>Your output should be similar to:</para>
+
+      <programlisting>NAME        TYPE         CLUSTER-IP    EXTERNAL-IP     PORT(S)         AGE
+eclservices ClusterIP    10.0.44.11    &lt;none&gt;          8010/TCP        11m
+<emphasis role="bold">eclwatch    LoadBalancer 10.0.21.16    12.87.156.228   8010:30190/TCP  11m</emphasis>
+kubernetes  ClusterIP    10.0.0.1      &lt;none&gt;          443/TCP         4h28m
+mydali      ClusterIP    10.0.195.229  &lt;none&gt;          7070/TCP        11m</programlisting>
+
+      <para>Use the EXTERNAL-IP listed for the ECLWatch service. Open a
+      browser and go to http://&lt;external-ip&gt;:8010/, for example in this
+      case, http://12.87.156.228:8010. If everything is working as expected,
+      the ECLWatch landing page displays.</para>
+    </sect2>
+
+    <sect2>
+      <title>Taking Down The AKS Cluster</title>
+
+      <para>Destroying the AKS Cluster will do just that - completely destroy
+      it. That is the Terraform term for taking down and removing all
+      resources and processes Terraform deployed.</para>
+
+      <para>Just as with the installation, the order that modules are
+      destroyed is also important. Keep in mind that <emphasis role="bold">the
+      AKS module must be destroyed before the Virtual network
+      module.</emphasis> Attempting to destroy the virtual network before the
```

Comment:
I don't think you need all the details about what may happen if you destroy things in the wrong order here. 
I suggest: 
Attempting to destroy resources in the wrong order could leave your deployment in an odd state and may incur unnecessary costs.